### ISSUE TITLE
[mesa 19.1.6]3 vulkan scalar cases failed due to timeout

### ISSUE THREAD

**Issue by renchenglei** [intel/external-mesa#130]
3 vulkan binding_model cases failed due to "timeout" on Android 10 with mesa 19.1.6.
```
dEQP-VK.ubo.random.scalar#62
dEQP-VK.ssbo.layout.random.scalar#19
dEQP-VK.ssbo.layout.random.scalar#75
```
**The above 3 issues also could be reproduced with mesa 19.0.6. With upstream master branch(e4a52bd65), following case could pass, need double check which could help fix following case on master branch.**
```
dEQP-VK.ssbo.layout.random.scalar#19
```
Steps to reproduce this issue(run with cts packages):
```
run cts -m CtsDeqpTestCases -t dEQP-VK.ubo.random.scalar#62
run cts -m CtsDeqpTestCases -t dEQP-VK.ssbo.layout.random.scalar#19
run cts -m CtsDeqpTestCases -t dEQP-VK.ssbo.layout.random.scalar#75
```

**Comment by tpalli** [comment]
For me these pass fine with Mesa master:

dEQP-VK.ubo.random.scalar#62
dEQP-VK.ssbo.layout.random.scalar#19

using KBL NUC. My guess is that these fail only with APL?

This one failed on KBL: dEQP-VK.ssbo.layout.random.scalar#75

**Comment by renchenglei** [comment]
@tpalli, yes, it is one APL specific issue, which may be caused by performance. APL is our Q4 target, we have to target at APL.
